lib/prometheus/business_collector.rb

```diff
@@ -84,6 +101,50 @@ def collect
     @client_policies.observe(
       Policy.where(account_id: client_accounts.select(:id)).count
     )
+
+    sq = Policy.joins(:profiles).select(:account_id, "profiles.ref_id").distinct
+    Policy.select('sq.ref_id', 'COUNT(sq.account_id)')
+          .from("(#{sq.to_sql}) as sq")
+          .group('sq.ref_id').each do |item|
+      @total_policies_by_account.observe(
+        item.attributes['count'], ref_id: item.attributes['ref_id']
+      )
+    end
+
+    sq = Policy.where(account_id: client_accounts.pluck(:id))
+               .joins(:profiles)
+               .select(:account_id, "profiles.ref_id").distinct
+    Policy.select('sq.ref_id', 'COUNT(sq.account_id)')
+          .from("(#{sq.to_sql}) as sq")
+          .group('sq.ref_id').each do |item|
+      @client_policies_by_account.observe(
+        item.attributes['count'], ref_id: item.attributes['ref_id']
+      )
+    end
+
+    sq = Policy.joins(:policy_hosts)
+               .joins(:profiles)
+               .select('policy_hosts.host_id', 'profiles.ref_id').distinct
+    Policy.select('sq.ref_id', 'COUNT(sq.host_id)')
+          .from("(#{sq.to_sql}) as sq")
+          .group('sq.ref_id').each do |item|
+      @total_policies_by_host.observe(
+        item.attributes['count'], ref_id: item.attributes['ref_id']
+      )
+    end
+
+    sq = Policy.where(account_id: client_accounts.pluck(:id))
+               .joins(:policy_hosts)
+               .joins(:profiles)
+               .select('policy_hosts.host_id', 'profiles.ref_id').distinct
+    Policy.select('sq.ref_id', 'COUNT(sq.host_id)')
+          .from("(#{sq.to_sql}) as sq")
+          .group('sq.ref_id').each do |item|
+      @client_policies_by_host.observe(
+        item.attributes['count'], ref_id: item.attributes['ref_id']
```

Comment:
I'd also observe/include average (and st. deviation) of hosts per policy type (profile ref_id).

Reply:
That we can do from grafana (I guess)